src/sdk/formatting/formatDate.ts

```diff
@@ -106,17 +108,21 @@ const defaultOptions = (options: DateFormatOptions): DateFormatOptions => {
 		options.now = new Date();
 	}
 
-	if (typeof(options.absolute) !== 'boolean') {
-		options.absolute = false;
+	if (typeof(options.allowRelative) !== 'boolean') {
```

Comment:
do you have a test that tests that this is set to true when allowRelative is not of type boolean?